### DISCUSSION TITLE
2D game example done with Needle

### DISCUSSION THREAD

**Discussion by FireDragonGameStudio** [needle-tools/needle-engine-support#124] [Demo and Starter Ideas]
Hi, as it was already stated in the forum, I'd like to request a 2D game example. This would (to a certain extent) require the creation of 2D collider components (and eventually camera configurations for pixel-perfect display). Spritesheet animation is already working, so a sample like "Super Needle Bros." or "The Legend of Needle" would be awesome. Thx :)

**Comment by hybridherbst** [comment]
Haha, **The Legend of Needle** sounds awesome! Thanks for the suggestion!

I believe the pixel perfect part is the main thing missing, most 2D games work just fine with regular 3D colliders and an orthographic or near-orthographic camera.
